### DISCUSSION TITLE
Building Uno Platform is very slow

### DISCUSSION THREAD

**Discussion by modula2019** [unoplatform/uno#15515] [Q&A]
Hi all,
Please what can I do to speed up building Uno platform solution in Visual Studio 2022. It takes close to  5 minutes a building a simple solution sometimes. I have compared building MAUI solution with UNO. I can see that MAUI builds very fast while UNO is very slow.
What could be responsible for this? Please advise me.

VS 2022 17.9.0
Uno Platform 5.0
.Net 8.0

Thanks

**Comment by jeromelaban** [comment]
Thanks for the report. Make sure to take a look at [this build performance article](https://platform.uno/docs/articles/guides/solution-building-single-targetframework.html#improve-your-own-solution) and let us know it this helps.

Once you've done what the article suggests, and it is still slow, can you [provide a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html)?
